srcpkgs/librime/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'librime'
 pkgname=librime
-version=1.5.3
-revision=4
+version=1.7.2
+revision=1
 build_style=cmake
 configure_args="-DCMAKE_BUILD_TYPE=None -DENABLE_LOGGING=OFF -DBUILD_TEST=ON"
 makedepends="boost-devel leveldb-devel marisa-devel opencc-devel
- yaml-cpp-devel gtest-devel"
+ yaml-cpp-devel gtest-devel capnproto-devel"
+hostmakedepends="pkg-config"
```

Comment:
You probably need `capnproto` in `hostmakedepends` as well. And they should go above `makedepends`

Reply:
Hmm, ARM still fails with those changes?